### PR TITLE
Add check does user have permission to get raw message and re-enable raw command

### PR DESCRIPTION
**Pull request by ks129**
Check is command invoker in `message.channel.members` (there is members who can see this channel) and when user is not, give feedback to user and return early. After this, I re-enabled command.

### CODE REVIEW

File: bot/exts/info/information.py [hunk 1]
```diff
@@ -419,10 +419,14 @@ def format_fields(self, mapping: Mapping[str, Any], field_width: Optional[int] =
         return out.rstrip()
 
     @cooldown_with_role_bypass(2, 60 * 3, BucketType.member, bypass_roles=constants.STAFF_ROLES)
-    @group(invoke_without_command=True, enabled=False)
+    @group(invoke_without_command=True)
     @in_whitelist(channels=(constants.Channels.bot_commands,), roles=constants.STAFF_ROLES)
     async def raw(self, ctx: Context, *, message: Message, json: bool = False) -> None:
         """Shows information about the raw API response."""
+        if ctx.author not in message.channel.members:
+            await ctx.send(":x: You can't get message from channel that you don't see.")
```
Comment (Xithrius) [inline bot/exts/info/information.py:427]:
```suggestion
            await ctx.send(":x: You do not have permissions to see the channel this message is in.")
```

**Review by Xithrius — CHANGES_REQUESTED** [review]
Looks good to me, just one change on the error message.

**Review by Xithrius — APPROVED** [review]
LGTM!

File: bot/exts/info/information.py [hunk 2]
```diff
@@ -419,10 +419,14 @@ def format_fields(self, mapping: Mapping[str, Any], field_width: Optional[int] =
         return out.rstrip()
 
     @cooldown_with_role_bypass(2, 60 * 3, BucketType.member, bypass_roles=constants.STAFF_ROLES)
-    @group(invoke_without_command=True, enabled=False)
+    @group(invoke_without_command=True)
     @in_whitelist(channels=(constants.Channels.bot_commands,), roles=constants.STAFF_ROLES)
     async def raw(self, ctx: Context, *, message: Message, json: bool = False) -> None:
         """Shows information about the raw API response."""
+        if ctx.author not in message.channel.members:
+            await ctx.send(":x: You do not have permissions to see the channel this message is in.")
```
Comment (Akarys42) [inline bot/exts/info/information.py:427]: Why not use an error embed here?

Comment (ks129) [reply bot/exts/info/information.py:427]: Many other error messages in this cog use same format.

Comment (Akarys42) [reply bot/exts/info/information.py:427]: That's a good point.

**Review by Akarys42 — APPROVED** [review]
LGMT!

**Pull request merged**